src/applications/model/gaming-streaming-server.cc

```diff
@@ -128,6 +102,60 @@ GamingStreamingServer::EraseStatistics (void)
   m_totSentPackets = 0;
 }
 
+void
+GamingStreamingServer::AddNewTraffic (Ptr<RandomVariableStream> packetSize,
+                                      Ptr<RandomVariableStream> interArrivalTime)
+{
+  NS_LOG_FUNCTION (this);
+  m_pksList.push_back (packetSize);
+  m_iatList.push_back (interArrivalTime);
+}
+
+void
+GamingStreamingServer::Send (Ptr<RandomVariableStream> packetSize,
+                             Ptr<RandomVariableStream> interArrivalTime)
+{
+  NS_LOG_FUNCTION (this);
+
+  uint32_t pktSize = packetSize->GetInteger ()+12; //TODO: think about 12
+
+  SeqTsHeader header;
+  header.SetSeq (m_seq++);
+  NS_ABORT_IF (pktSize < header.GetSerializedSize ());
+  Ptr<Packet> packet = Create<Packet> (pktSize - header.GetSerializedSize ());
+  packet->AddHeader (header);
+  NS_ABORT_IF (packet->GetSize () != pktSize);
+
+  m_socket->Send (packet);
+  m_totSentPackets++;
+
+  if (Ipv4Address::IsMatchingType (m_peerAddress))
+    {
+      NS_LOG_INFO ("At time " << Simulator::Now ().GetSeconds ()
+                              << "s gaming server sent "
+                              <<  packet->GetSize () << " bytes to "
+                              << Ipv4Address::ConvertFrom(m_peerAddress)
+                              << " port " << m_peerPort);
+    }
+  else if (Ipv6Address::IsMatchingType (m_peerAddress))
+    {
+      NS_LOG_INFO ("At time " << Simulator::Now ().GetSeconds ()
+                              << "s gaming server sent "
+                              <<  packet->GetSize () << " bytes to "
+                              << Ipv6Address::ConvertFrom(m_peerAddress)
+                              << " port " << m_peerPort);
+    }
+  // Schedule next packet of this stream
+  Simulator::Schedule (MilliSeconds (interArrivalTime->GetInteger ()),
```

Comment:
for improved debugging I would create the iat outside and give it as argument.
Also, print it as debug info